.github/workflows/pull-request.yml

```diff
@@ -2,6 +2,7 @@ name: PR Build Check
 
 on:
   pull_request:
+    types: [labeled, unlabeled, opened, synchronize, reopened]
```

Comment:
wouldn't this cause also for the main job to run too simply by adding labels?

Reply:
Nope, that's only for PRs